backend/server/src/http.rs

```diff
@@ -184,6 +185,11 @@ async fn handle(req: Request<Body>, ctx: Arc<Context>) -> Result<Response, hyper
             serve_index().await
         }
 
+        // The player page
+        (&Method::GET, path) if path.starts_with(PLAYER_PREFIX) => {
+            serve_index().await
+        }
+
```

Comment:
Mhhhh this doesn't scale. We still need to think of a good way to not repeat all the routes in the backend, at least not with so much duplicated code. But that doesn't have to be solved by this PR. Just thoughts.

Reply:
That's what I thought as well!! :grimacing: